kit/blueprint/styles.scss

```diff
@@ -260,6 +260,22 @@ textarea.bp3-input,
   box-shadow: var(--xh-form-field-box-shadow);
 }
 
```

Comment:
I'm sure you've tried, but it would be great if we can get the below styles working *without* the high specificity added by the `body.xh-app` wrappers. We have a similar setup with our button styling, and it leads to a kind of arms-race of highly specific rules whenever you want to override the core style in a component or app.

Reply:
Seems to work today with body.xh-app removed.   Not sure how got lucky today or unlucky before.